pages/expectations.tsx

```diff
@@ -53,7 +54,54 @@ const Expectations: FC = () => {
         <p className="m-0">{t('do-not-travel')}</p>
       </blockquote>
       <h2 className="h2">{t('header-privacy')}</h2>
-      <p>{t('description-privacy')}</p>
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.1'}>
+          Insert feedback{' '}
+          <ExternalLink href={t('expectations:description-privacy.1.1-link')}>
+            Link
+          </ExternalLink>
+        </Trans>
+      </p>
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.2'}>
+          Insert feedback{' '}
+          <ExternalLink href={t('expectations:description-privacy.2.2-link')}>
+            Link
+          </ExternalLink>
+        </Trans>
+      </p>
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.3'}>
+          Insert feedback{' '}
+          <ExternalLink href={t('expectations:description-privacy.3.3-link')}>
+            Link
+          </ExternalLink>
+        </Trans>
+      </p>
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.4'}>
```

Comment:
Use `Trans` component property `components` as mentionned in a previous comment for keys 4, 5 and 6.

Link key `description-privacy.4.4-link` is wrong. It should be `description-privacy.4-link`. Same for the other keys.